game-core/src/main/java/games/strategy/triplea/ui/screen/TileManager.java

```diff
@@ -567,6 +567,20 @@ public void setTerritoryOverlay(
     updateTerritory(territory, data, mapData);
   }
 
+  public void setTerritoryOverlayForTile(
+      final Territory territory,
+      final Color color,
+      final int alpha,
+      final GameData data,
+      final MapData mapData) {
+    synchronized (mutex) {
+      final IDrawable drawable =
+          new TerritoryOverLayDrawable(color, territory.getName(), alpha, Operation.FILL);
```

Comment:
I thought `setTerritoryOverlay` would be what I wanted but it only colors the border of the territory.  And that isn't very visible.  So, this new method uses the `FILL` operation so that the entire territory is colored.